python/pysmurf/client/command/smurf_command.py

```diff
@@ -410,18 +410,18 @@ def set_defaults_pv(self, **kwargs):
         """
         Sets the default epics variables
         """
-        self._caput(self.epics_root + ':AMCc:setDefaults', 1, wait_after=20,
+        self._caput(self.epics_root + ':AMCc:setDefaults', 1, wait_after=30,
             **kwargs)
         self.log('Defaults are set.', self.LOG_INFO)
 
 
     def set_read_all(self, **kwargs):
         """
         ReadAll sends a command to read all register to the pyrogue server
-        Registers must upated in order to PVs to update.
-        This call is necesary to read register with pollIntervale=0.
+        Registers must updated in order to PVs to update.
+        This call is necessary to read register with pollIntervale=0.
         """
-        self._caput(self.epics_root + ':AMCc:ReadAll', 1, wait_after=5,
+        self._caput(self.epics_root + ':AMCc:ReadAll', 1, wait_after=20,
```

Comment:
Do we understand why we need 4x as long a wait?

Reply:
I found this value empirically using the GUI. I execute the ReadAll command and then measure how long does it take to the GUI to start updating values again. The EPICS server keeps responding during this time, but I think it is better to increase this wait to make sure all the registers have been updated by the `ReadAll` command before continuing.

Reply:
@jesusvasquez333 @eyyoung24 @slacrherbst  is it possible to have ReadAll set some flag when it completes instead of having an arbitrary 20 sec wait?  I think this would be a more robust solution if possible, since who can say if this time gets less or more in later releases...